### DISCUSSION TITLE
NAT forwarding

### DISCUSSION THREAD

**Discussion by arobase13** [modoboa/modoboa#2760] [Help]
Hello,
I installed a few days ago Modoboa on a server at home under Debian 11.
On my router I redirected to ports 25, 80, 443, 993, 587 and 143.
I can access my webmail remotely without any problem, the same goes for reception.

On the other hand, from thunderbird type software, I have to be connected to the same network as my server.

There is still a port forwarding to do.

Thank you in advance for your help.

Manu

**Comment by tonioo** [comment]
@arobase13 When does it fail? To retrieve messages or send them?
